main.nf

```diff
@@ -254,11 +256,19 @@ if (params.library == 'classic'){
     library = '--UDGhalf'
 }
 
-if( ! nextflow.version.matches(">= 0.30") ){
+if( ! nextflow.version.matches(">= 19.04.0") ){
     println "Your version of Nextflow is too old, please update to Nextflow >= 0.30"
     exit(1)
 }
 
+// Check sourcepredict parameters
+
+if (params.sp_embed != 'mds' && params.sp_embed != 'tsne' && params.sp_embed != 'umap'){
+    println "${params.sp_embed} is not a valid method for SourcePredict embedding"
+    println "Available methods are: mds, tsne, umap"
```

Comment:
Small suggestion: as nextflow best practise is to put the strings in quotes on the command itself, maybe put the quotes around each moethod in line 268

Reply:
Done 8d97a46